docs/content/tools/_index.md

```diff
@@ -11,9 +11,9 @@ toc:
   enabled: true
 ---
 
-The [OSCAL models](/concepts/layer/) provide standardized formats for exchanging control, control implementation, and control assessment information in XML, JSON, and YAML, in support of interoperable security assessment automation and continuous monitoring. These formats allow this information to be exchanged between tools ensuring interoperability, and for individual tools, to process exchanged data, supporting analytics, user interaction, and security assessment automation where suitable.
+Please see  [Awesome OSCAL](https://github.com/oscal-club/awesome-oscal) for a list of tools maintained by the OSCAL community.
```

Comment:
It seems we have just moved the link slightly higher, above the current callout text. How do we know this will address that one user's confusion, or that of others?